tutorials/lesson1-booleans/booleans.qnt

```diff
@@ -1,42 +1,34 @@
-/**
- This lesson teaches you the basics of operations over Booleans.
- If you have programming experience, you know all of these operators.
- So it should not take you long to finish this lesson.
-
- If you do not know how to start the lesson, do the following:
-
-  - Open Visual Studio Code.
-  - Install the Quint and Codetour extensions.
-  - Choose the tour "Lesson 1 - Booleans".
+/*
+ If you do not know how to start this lesson, see:
+ https://github.com/informalsystems/quint/tree/main/tutorials
  */
```

Comment:
Is this alignment right on line 1?

Reply:
Yep, looks like wrong indentation. Fixed